README.elk.md

```diff
@@ -0,0 +1,143 @@
+
+```
+      {
+        "name": "Elasticsearch",
+        "shortname": "elasticsearch",
+        "enable_auto_healing": true,
+        "install_updates_on_boot": true,
+        "type": "custom",
+        "auto_assign_public_ips": false,
+        "use_ebs_optimized_instances": true,
+        "custom_recipes": {
+            "setup": [
+                "mh-opsworks-recipes::set-timezone",
+                "mh-opsworks-recipes::fix-raid-mapping",
+                "mh-opsworks-recipes::set-bash-as-default-shell",
+                "mh-opsworks-recipes::install-utils",
+                "mh-opsworks-recipes::enable-postfix-smarthost",
+                "mh-opsworks-recipes::install-custom-metrics",
+                "mh-opsworks-recipes::create-alerts-from-opsworks-metrics",
+                "mh-opsworks-recipes::enable-enhanced-networking",
+                "mh-opsworks-recipes::install-elasticsearch",
+                "mh-opsworks-recipes::clean-up-package-cache"
+            ],
+            "shutdown": [
+                "mh-opsworks-recipes::remove-alarms"
+            ]
+        },
+        "volume_configurations": [{
+            "mount_point": "/vol/data",
+            "number_of_disks": 1,
+            "size": "50",
+            "volume_type": "gp2"
+        }],
+        "instances": {
+            "number_of_instances": 1,
+            "instance_type": "t2.medium",
+            "root_device_type": "ebs"
+        }
+      },
+      {
+        "name": "Logstash-Kibana",
+        "shortname": "logstash-kibana",
+        "enable_auto_healing": true,
+        "install_updates_on_boot": true,
+        "type": "custom",
+        "auto_assign_elastic_ips": true,
+        "auto_assign_public_ips": true,
+        "use_ebs_optimized_instances": false,
+        "custom_recipes": {
+            "setup": [
+                "mh-opsworks-recipes::set-timezone",
+                "mh-opsworks-recipes::fix-raid-mapping",
+                "mh-opsworks-recipes::set-bash-as-default-shell",
+                "mh-opsworks-recipes::install-utils",
+                "mh-opsworks-recipes::enable-postfix-smarthost",
+                "mh-opsworks-recipes::install-custom-metrics",
+                "mh-opsworks-recipes::create-alerts-from-opsworks-metrics",
+                "mh-opsworks-recipes::enable-enhanced-networking",
+                "mh-opsworks-recipes::install-logstash-kibana",
+                "mh-opsworks-recipes::clean-up-package-cache"
+            ],
+            "configure": [
+                "mh-opsworks-recipes::configure-logstash-kibana",
+                "mh-opsworks-recipes::install-ua-harvester"
+            ],
+            "shutdown": [
+                "mh-opsworks-recipes::remove-alarms"
+            ]
+        },
+        "volume_configurations": [
+
+        ],
+        "instances": {
+            "number_of_instances": 1,
+            "instance_type": "t2.medium",
+            "root_device_type": "instance-store"
```

Comment:
Well, because we're defaultly using ebs-backed instances, logs under /var/log *are* persistent and stored on EBS volumes. I am definitely willing to experiment, though.